api/file.go

```diff
@@ -174,47 +170,23 @@ func fireAndForgetHandleImages(filenames []string, fileData [][]byte, teamId, ch
 				// Get the image's orientation and ignore any errors since not all images will have orientation data
 				orientation, _ := getImageOrientation(fileData[i])
 
-				// Create a temporary image that will be manipulated and then used to make the thumbnail and preview image
-				var temp *image.RGBA
 				switch orientation {
-				case Upright, UprightMirrored, UpsideDown, UpsideDownMirrored:
-					temp = image.NewRGBA(img.Bounds())
-				case RotatedCCW, RotatedCCWMirrored, RotatedCW, RotatedCWMirrored:
-					bounds := img.Bounds()
-					temp = image.NewRGBA(image.Rect(bounds.Min.Y, bounds.Min.X, bounds.Max.Y, bounds.Max.X))
-
-					width, height = height, width
-				}
-
-				// Draw a white background since JPEGs lack transparency
-				draw.Draw(temp, temp.Bounds(), image.NewUniform(color.White), image.Point{}, draw.Src)
```

Comment:
We still need something like this line to draw a white background behind the image in case it's transparent. Without that, it ends up with a black background behind the preview and thumbnail. The rest of this looks good though.